projects/igniteui-angular/src/lib/tree/tree-navigation.spec.ts

```diff
@@ -301,6 +302,70 @@ describe('IgxTree - Navigation #treeView', () => {
             });
         });
 
+        describe('UI Interaction tests - Expand/Collapse nodes', () => {
```

Comment:
Very well-described and structured unit tests, one minor improvement that you might consider, we can try moving the  common logic on every `it` execution, for the firstNode _to be collapsed_ check, into a common function like firstNodeSetup and performs the assigning and firstNode and expanded to be falsy assert, just a thought..

```
    function firstNodeSetup(toggleNodeOnClick: boolean) {
        tree.toggleNodeOnClick = toggleNodeOnClick;
        fix.detectChanges();
        const firstNode = tree.nodes.toArray()[0];
        expect(firstNode.expanded).toBeFalsy();
        return firstNode;
    }
```